frontend/utils.mjs

```diff
@@ -92,17 +97,25 @@ export function copyIndexHtml(
         window.ESBUILD_LOAD_CHUNKS('index');
     `
 
-    const cssLinkTag = cssFile ? `<link rel="stylesheet" href='${useJsURL ? jsURL : ''}/static/${cssFile}'>` : ''
+    // Snippet to dynamically load the css based on window.JS_URL
+    const cssLoader = `
+        const link = document.createElement("link");
+        link.rel = "stylesheet";
+        link.href = window.JS_URL + "static/${cssFile}"
```

Comment:
Qq: should `static/` be hardcoded? It was before so I think it is fine for now

Reply:
Yeah my reason for leaving is just to not change anything other than the required. I'd rather get staging a reproduction of production, get that workflow going, then we can consider how we can improve.